### ISSUE TITLE
将情感分类改为三分类时遇到expected dense_1 to have 2 dimensions, but got array with shape (32, 1, 3)

### ISSUE THREAD

**Issue by xxxxxxxxy** [bojone/bert_in_keras#6]
`np_utils.to_categorical(np.squeeze(y, axis=-1), NB_CLASSES)`
```
model.compile(
    loss='categorical_crossentropy',
    optimizer=Adam(1e-5), # 用足够小的学习率
    metrics=['accuracy']
)
```
